rosidl_generator_cpp/resource/idl__struct.hpp.em

```diff
@@ -29,6 +29,7 @@ include_directives = set()
 #include <string>
 #include <vector>
```

Comment:
Would the include of vector now be superfluous?

Reply:
Good point.